### DISCUSSION TITLE
Welcome to MGT2_Mod_Tool Discussions!

### DISCUSSION THREAD

**Discussion by LMH01** [LMH01/MGT2_Mod_Tool#19] [General]
**In this place you can share your exports with other players**
If you would like to submit something just create a new discussion and write what your "mods" will add. Then link to a file sharing service where others can download your exports. Please use google drive or dropbox.

**Comment by Joti17** [comment]
Hallo,
ich wollte fragen, ob du das Limit für Engine Features und Gameplay Features für das Tech Level etc. ändern kannst und ich habe einen Spielstand, aber ich kann dort keine neuen mods hinzufügen
Mit Freundlichen Grüssen
Joti17

**Reply by LMH01** [reply]
Das Limit kann umgangen werden, indem die Option "unlock spinners" in den Einstellungen unter "configure safety features" eingeschaltet wird.

**Comment by Joti17** [comment]
Kannst du eine Ultra op Version veröffentlichen, wo es keine limets gibt, und ich die zahl bei z.B. Tech reinschreiben kann und nicht die ganze Zeit denn hoch knopf drücken muss

**Reply by LMH01** [reply]
Siehe Antwort zu der ersten Frage.
